### ISSUE TITLE
What's the Accuracy ?

### ISSUE THREAD

**Issue by Zozoz** [ematvey/deep-text-classifier#1]
Could you tell me the acc in the Yelp-2013/2014/2015 by running your code? I run the code, but I could not reach the acc written in the paper.

Thanks!

**Comment by ematvey** [comment]
Just finished fresh train cycle. Accuracy is 65.08% on a dev set (10%). I did not do any hyperparameter tuning.
